obspy/io/sac/LICENSE.txt

```diff
@@ -0,0 +1,16 @@
+Copyright 2015. Los Alamos National Security, LLC for pysac LA-CC-15-051. This
+material was produced under U.S. Government contract DE-AC52-06NA25396 for Los
+Alamos National Laboratory (LANL), which is operated by Los Alamos National
+Security, LLC for the U.S. Department of Energy. The U.S. Government has rights
+to use, reproduce, and distribute this software.  NEITHER THE GOVERNMENT NOR
+LOS ALAMOS NATIONAL SECURITY, LLC MAKES ANY WARRANTY, EXPRESS OR IMPLIED, OR
```

Comment:
If it's LGPL we don't need a License file in this submodule. The copyright information could be moved to the README.txt and/or `__init.__.py`, I think.